RPC edits (#1)
rfc-editor/AUTH48-rfc9920: rfc9920.md

{{RFC9280}} contained significant changes to the publication model for RFCs.
Those changes created new structures and new processes for the publication of RFCs.
As these structures and processes have been exercised, the community has found places where they might be improved.
In addition, gaps in some of the processes have been found.
This document updates {{RFC9280}} based on these findings.

The organization of this RFC is different from typical RFCs in order to keep the section numbering the same as {{RFC9280}}.
To keep the section numbering the same, the Introduction section is much longer, with several subsections that refer to the main body.

The remainder of this introduction is a list of changes to {{RFC9280}}.
Those changes are instantiated in the rest of the document, with cross-references between the list of changes and the main body.

## RPC Roles and Responsibilities

{{RFC9280}} created a new structure for the RFC Editor function. It established the RFC Series Working Group (RSWG) and the RFC Series Approval Board (RSAB) and gave new responsibilities to the RFC Production Center (RPC).

> Broadly speaking, it says that the RSWG writes policies for the Editorial Stream, the RSAB approves those policies, and the RPC implements those policies.

Reviewer: Nit.  Do documents "say"?

```suggestion
In broad terms, the RSWG writes policies for the Editorial Stream, the RSAB approves those policies, and the RPC implements those policies.
```

Reviewer 2: @paulehoffman and @alexisannerossi, what are your thoughts on this suggestion from Eliot? Note that "says" has been used in this type of context in published RFCs. Also note that this document contains other instances of "says" (e.g., "Section X says") aside from this instance and the other that Eliot comments on below.

Reviewer 3: "Says" is fine because you use it in other documents. We trust your editorial judgement here.

>
> {{intent}} introduces the term "consumers of RFCs", referring to them as "constituent stakeholders" who should be considered by the RSAB when approving Editorial Stream policy documents.
>
> "Consumers of RFCs" is now defined to mean those people who read RFCs to understand, implement, critique, and research the protocols, operational practices, and other content as found in RFCs.
> 
> The policy to be followed by the RFC publication streams and RFC Editor in respect to consumers of RFCs is as follows:
>
> - Consumers of RFCs MUST be considered as separate constituent stakeholders from IETF/IRTF participants.
While IETF/IRTF participants and others involved in the development and production of RFCs may be consumers of RFCs, the two are distinct, overlapping sets.
>
> - The [RFC Editor website](https://www.rfc-editor.org) MUST be primarily focused on consumers of RFCs.
>
> - Consumers of RFCs MUST NOT be required or expected to become IETF/IRTF participants unless they wish to extend, update, or modify an RFC.

## Updates to RFC 9720

{{RFC9720}}, "RFC Formats and Versions", updated {{RFC9280}}.

### RFCs May Be Reissued {#reissued}

> {{Section 7.6 of RFC9280}} says:

Reviewer: Same point as to the above.

```suggestion
{Section 7.6 of RFC9280}} states:
```
